src/pytesseract.py

```diff
@@ -176,7 +177,7 @@ def prepare(image):
     if 'A' in image.getbands():
         # discard and replace the alpha channel with white background
         background = Image.new(RGB_MODE, image.size, (255, 255, 255))
-        background.paste(image, (0, 0), image)
+        background.paste(image, (0, 0), image.convert(RGBA_MODE))
```

Comment:
Since we are trying to strip the alpha channel, shouldn't we use `image.convert(RGB_MODE)` instead?
We already have the background and we just need to overwrite the R, G and B channels, right?